src/components/Product_card/ProductCard.tsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+
+interface ProductCard_props {
+  name: string;
+  brand: string;
+  product_image?: { url: string };
+  price: number;
+  id: number;
+}
+
+export const ProductCard = ({
+  name,
+  brand,
+  price,
+  id,
+  product_image,
+}: ProductCard_props) => {
+  <div className="card_cont">
```

Comment:
Don't use classNames or css, Styled Component 
```suggestion
  <Root>
```